docs/Token_Assumptions.md

```diff
@@ -0,0 +1,21 @@
+_Attestation Token use cases & assumptions_
+The following captures what is assumed about the nature of the attestation tokens received for verification.
+
+A token is presented as a single 'blob' (byte stream).
+
+Tokens are presented individually. If a batch mode submission is required, that is the responsibility of an API tier.
+
+Encoding in transit is the responsibility of the API tier, the verification pipeline works on a byte stream.
+
+The verification pipeline will receive some context information which will indicate or help to deduce the nature of the token and hence which plugins will be needed in the verification pipeline. In a dedicated deployment this may be hardwired, in an environment supporting multiple token types, this is likely to come from the API tier.
+
+A token may have an encrypted body. Assuming the correct decryption key is available, the body will be decrypted and then passed into the pipeline in the same manner as an unencrypted token.
```

Comment:
I think the fact that the token was encrypted should be passed on to the verification step because this may be relevant to the verification policy.  (E.g., for privacy considerations)

Reply:
Agree that noting it was encrypted is a useful piece of state, but I wanted to capture that encrypted tokens are supported and that decrypting them is seen as a separate operation.